packages/reports/addon/consumers/request/column.ts

```diff
@@ -59,6 +59,7 @@ export default class ColumnConsumer extends ActionConsumer {
       const { routeName } = route;
       const { request } = route.modelFor(routeName) as ReportModel;
       request.removeColumn(column);
+      this.requestActionDispatcher.dispatch(RequestActions.REMOVE_ROLLUP_COLUMN, route, column);
```

Comment:
Is this already handled here:
https://github.com/yavin-dev/framework/blob/94f19a411c94502083422b09e15792da742d2a6e/packages/reports/addon/consumers/request/rollup.ts#L56

Reply:
Bad rebase, i'll fix.